app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -77,9 +77,9 @@ const MakeDecision = ({ data }) => {
     selectedItems: [],
   });
   const [tabs, toggleTab, collapseTabsExceptCurr] = useAccordionTabsState({
-    [MACHINE_LEARNING_SUGGESTIONS]: true,
+    [MACHINE_LEARNING_SUGGESTIONS]: isAnalyzerAvailable,
     [COPY_FROM_HISTORY_LINE]: false,
-    [SELECT_DEFECT_MANUALLY]: false,
+    [SELECT_DEFECT_MANUALLY]: !isAnalyzerAvailable,
```

Comment:
select defect type manually doesn't  dependent from analyzer and should be open by default